lib/utils.js

```diff
@@ -490,6 +501,12 @@ api.verifyCapabilityChain = async ({
       throw error;
     }
 
+    // if root.expires is undefined then Date.parse returns NaN which is falsey
+    const rootExpirationDate = Date.parse(root.expires);
```

Comment:
Is there a concern with bogus data here (and with the other parse())?  Passing in any unknown format will result in NaN.  Would checking that the parse call worked be too strict?  Or maybe just require the date fields are validated before any of this code is called.

Reply:
Attempted to address: 585228e46ef73c99ce7a2c436000ac9a20714e04